multimodal/src/autogluon/multimodal/utils/config.py

```diff
@@ -812,3 +813,59 @@ def update_ensemble_hyperparameters(
         hyperparameters = presets_hyperparameters
 
     return hyperparameters
+
+
+def make_overrides_backward_compatible(overrides: Dict):
```

Comment:
should we do a `overrides = copy.deepcopy(overrides)` at the start of this function to avoid in-place mutation of the user's dictionary?

Reply:
A `copy.deepcopy` is applied before calling function `apply_omegaconf_overrides`, so no need to copy them again. See line 188 here: https://github.com/autogluon/autogluon/pull/4737/files#diff-a10c8783a894991a02709f19d3b1a2653731c04d798ec8b5234b0c9dc2d55bcdR188